tests/data/biasRun.yaml

```diff
@@ -1,16 +1,21 @@
 2021052500015:
+  SUCCESS: false
```

Comment:
The values in the results dictionary show this is getting a value of ~125, so there's a bug somewhere.  DM-47023 is the ticket to track down this change.